src/org/omegat/Bundle.properties

```diff
@@ -1769,9 +1769,9 @@ SEG_NEW_LN_CO=New language and country
 # TeamOptions
 PREFS_TITLE_TEAM=Team
 
-GUI_TEAM_DESCRIPTION=Your name/ID is used to identify your translations.
+GUI_TEAM_DESCRIPTION=This nickname will be used to identify your translations in the project TMX.
 
-WF_OPTION_INSERT_AUTHOR_PREFIX=&Name/ID:
+WF_OPTION_INSERT_AUTHOR_PREFIX=Nickname:
```

Comment:
I'm not entirely sure about "nickname". It's not wrong, but something about it bugs me a little, though I can't quite put my finger on why. Maybe "alias" would be better?

Given the context, I think line 1772 should probably be:
`Enter the alias you want to use to identify your translations in the memory.`

("Enter the nickname…" if we decide to keep that)

Line 1774 then either becomes `Alias:` or stays as is.

Interestingly, this alias/nickname applies even in non team projects, so this entire section should perhaps be under the `General` heading, but that's beyond the scope of this specific PR.